app/models/iiif_content_search_response.rb

```diff
@@ -108,6 +118,18 @@ def text
       highlight[first...last].gsub(%r{</?em>}, '')
     end
 
+    def before
+      first = highlight.index('<em>')
+      highlight[0...first].split.map { |x| split_word_and_payload(x) }.map(&:first).join(' ')
```

Comment:
Would we get any benefit by calling `split_word_and_payload(x).first` in order to avoid the additional `map`?